package.json

```diff
@@ -59,12 +59,20 @@
     "pug-cli": "1.0.0-alpha6",
     "should": "11.1.0",
     "sinon": "1.17.5",
-    "webpack": "1.13.2"
+    "webpack": "1.13.2",
+    "jade": "1.11.0",
```

Comment:
Rendering engine was migrated to pug in #555, could we please remove the legacy `jade` dep?